artifacts/scripts/util.sh

```diff
@@ -776,6 +776,9 @@ function reset-godeps() {
         git rm -rf Godeps
     fi
 
+    git config user.email "$GIT_COMMITTER_EMAIL"
```

Comment:
can't we set this once?

Reply:
i could not figure out where we could. any suggestions? i threw them in only when they failed. so took multiple iterations to get this set of changes

Reply:
Why not move it to https://github.com/kubernetes/publishing-bot/blob/master/artifacts/scripts/construct.sh#L83 ?

Reply:
this is called from construct.sh only. No need for these two lines.